protostar/commands/test/test_command.py

```diff
@@ -97,6 +97,11 @@ def arguments(self) -> List[Command.Argument]:
                 type="bool",
                 description="Exit immediately on first broken or failed test",
             ),
+            Command.Argument(
+                name="stdout-on-success",
+                type="bool",
+                description="Also print captured stdout for passed test cases.",
```

Comment:
```suggestion
                description="Print captured standard output for passed test cases.",
```

Reply:
This "also" is here to show that the default behaviour is that we always print for failed test cases